app/models/registration/EuTaxRegistration.scala

```diff
@@ -61,9 +65,37 @@ object RegistrationWithFixedEstablishment {
     Json.format[RegistrationWithFixedEstablishment]
 }
 
-final case class RegistrationWithoutFixedEstablishment(country: Country) extends EuTaxRegistration
+
+final case class RegistrationSendingGoods(
+                                           country: Country,
+                                           taxIdentifier: EuTaxIdentifier,
+                                           sendsGoods: Boolean,
```

Comment:
Do we need to store the booleans? I think that data is implied by the model - i.e. WithFixedEstablishment won't send goods, WithoutFixedEstablishment does _if_ the new object I mentioned above is provided